tests/perf/configuration/configuration_test.go

```diff
@@ -0,0 +1,268 @@
+//go:build perf
+// +build perf
+
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package configuration
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/dapr/dapr/tests/perf/utils"
+	kube "github.com/dapr/dapr/tests/platforms/kubernetes"
+	"github.com/dapr/dapr/tests/runner"
+	"github.com/dapr/dapr/tests/runner/loadtest"
+	"github.com/dapr/dapr/tests/runner/summary"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	tr *runner.TestRunner
+)
+
+const (
+	numHealthChecks                   = 60 // Number of times to check for endpoint health per app.
+	defaultConfigGetThresholdMs       = 500
+	defaultConfigSubscribeThresholdMs = 1000
```

Comment:
based on the latencies stated [here](https://github.com/dapr/dapr/pull/6247#issuecomment-1511028152), why do we have such high buffer values?